### ISSUE TITLE
Rotating the device and back causes the video to stall (but it's actually still playing) 

### ISSUE THREAD

**Issue by scgough** [TheWidlarzGroup/react-native-video#1953]
# Bug

If I'm watching a video in portrait and I rotate my device to landscape, it continues playing. 
However, if I then rotate it back to portrait the video (visually) stalls but from the native code side it looks like it is still playing in the background. 

- Pausing and un-pausing the video when this happens does nothing to fix it.
- Seeking the video back to `0` index does nothing to fix the issue. 

## Platform
* iOS
* Android ExoPlayer
* (using native controls when in landscape, no controls in portrait)

## Environment info
RN 0.59.10
React 16.8.6
react-native-video 5.1.0-alpha1

## Steps To Reproduce
Tested on iOS sim and real iOS and Android device 

Play video in portait from a URL source. 
While playing back, turn your device to landscape (video still plays)
Turn your device back to portrait (video stalls)

On the native side I can see `playerItemDidReachEnd` firing with an `NSLog` output strategically placed. 

## Expected behaviour
Rotating the device should not affect video playback.

**Comment by iacocesar** [comment]
same

**Comment by iacocesar** [comment]
any solution @scgough ?

**Comment by scgough** [comment]
@IacoCesar - This was a bit of a strange one. I had to _bypass_ it in the end. 

It turns out it was seemingly down to the fact there was a sliding panel which sometimes had additional videos playing within the portrait version of the screen. The additional video component instances seemed to affect playback of the 'main' component. 

We switched the additional videos to thumbnail images and the playback now works between screen rotations.
